publish.js

```diff
@@ -8,7 +8,7 @@ const readline = require("readline")
 
 const rl = readline.createInterface({
     input: process.stdin,
-    output: process.stdout
+    output: process.stdout,
```

Comment:
This seems to be the result of using a different prettier config or version. Please align your local settings with this repo, or don't stage the unrelated files, or creating a PR that locks the config and prettier version would be even more awsome!